src/core/agent/services/multiSigService.test.ts

```diff
@@ -243,7 +243,8 @@ describe("Oobi/endrole", () => {
       otherIdentifiers.length + 1
     );
     expect(addEndRoleMock).toBeCalledTimes(1);
-    (memberMetadataRecord.groupMetadata as any).groupCreated = false;
+    if(memberMetadataRecord.groupMetadata)
```

Comment:
What does this line actually do?

Reply:
it will violate @typescript-eslint/no-non-null-assertion rule.

Reply:
I'm just wondering if this line is necessary for the test, since it's at the end of the test